docker.go

```diff
@@ -49,6 +49,11 @@ const (
 	packagePath   = "github.com/testcontainers/testcontainers-go"
 
 	logStoppedForOutOfSyncMessage = "Stopping log consumer: Headers out of sync"
+
+	healthStatusNone      = ""          // default status for a container with no healthcheck
```

Comment:
hello @emetsger ! up to you, can use here status from[ docker/types](https://github.com/moby/moby/blob/9314eaff2f5771ffc3843961e50bdd5cedfa34c8/api/types/types.go#L254)?

who knows maybe docker change name status in the future:)

Reply:
👍 

`docker/types` defines `types.NoHealthcheck = none`.  I did not see a constant for a health check value equal to the zero-value string.

So, to be safe, I check for a zero-length string before checking the health status value:
```go
// if a health status is present on the container, and the container is not healthy, error
if r.healthStatus != "" {
	if r.healthStatus != types.Healthy && r.healthStatus != types.NoHealthcheck {
		return fmt.Errorf("container %s is not healthy, wanted status=%s, got status=%s", resp[0].ID[:8], types.Healthy, r.healthStatus)
	}
}
```

Reply:
yeah, agree:) thanks!